src/python/pose_format/numpy/pose_body.py

```diff
@@ -339,13 +339,18 @@ def interpolate(self, new_fps: int = None, kind='cubic'):
         for people in points:
             new_frames = []
             for frames in people:
-                mask = frames.transpose()[0].mask
+                mask = frames.transpose()[-1].mask
```

Comment:
The 0 dim is the x, and the -1 is the confidence.

I somehow get some pose data that has some x, y, z, and confidence all 0s (but not masked), so here reading the confidence for the mask is more reliable since it is set explicitly on lines 334 and 335.

Reply:
ah! ok, can you please add a comment. `# takes mask from confidence value` to make it clear?